guardian/Makefile

```diff
@@ -69,9 +69,12 @@ $(GUARDIAN_CONTAINER_CREATED): docker-image/guardian/Dockerfile $(BINDIR)/guardi
 #############################################
 .PHONY: ut
 ## Run only Unit Tests.
-ut:
+ut: generate-test-certs
 	$(DOCKER_GO_BUILD) go test ./... -cover -count 1
 
+generate-test-certs:
```

Comment:
We should move away from anything that's generating certs like this for testing, we can create them in golang as the test needs it.